syncd/Makefile.am

```diff
@@ -20,6 +20,12 @@ if SAICAVM
 SAIFLAGS = -DCAVMSAI
 endif
 
+if SAIVS
+SAILIB=-L$(top_srcdir)/vslib/src/.libs -lsaivs
```

Comment:
Not sure about this, i will need to investigate on that, for now i leave this as is since its working